test/infrastructure/docker/api/v1beta1/dockermachinetemplate_webhook_test.go

```diff
@@ -32,11 +38,14 @@ func TestDockerMachineTemplateInvalid(t *testing.T) {
 
 	newTemplate := oldTemplate.DeepCopy()
 	newTemplate.Spec.Template.Spec.ExtraMounts = append(newTemplate.Spec.Template.Spec.ExtraMounts, []Mount{{ContainerPath: "/var/run/docker.sock", HostPath: "/var/run/docker.sock"}}...)
+	newTemplateSkipImmutability := oldTemplate.DeepCopy()
```

Comment:
```suggestion
	newTemplateSkipImmutability := newTemplate.DeepCopy()
```

I think this only actually proves we skipped the test if the spec of newTemplateSkipImmutability is different than oldTemplate which it currently isn't